### PR TITLE
Report diagnostic counters as Delta Counters

### PR DESCRIPTION
**Pull request by joannak-vmware**

### CODE REVIEW

**Review by haosong — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -209,7 +209,6 @@ wavefront_sender = WavefrontProxyClient(
    distribution_port=2878,
    tracing_port=30000,
    event_port=2878
-   internal_flush=2,
```
Comment (hanwavefront) [inline README.md:212]: did you mean to remove this (or maybe `event_port=2878` needed a comma)? if you are removing it, also remove `# a nondefault interval (2 seconds) for flushing data from the sender to the proxy. Default: 5 seconds` comment above.

**Review by hanwavefront — CHANGES_REQUESTED** [review]
one minor detail, otherwise LGTM

File: setup.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 setuptools.setup(
     name='wavefront-sdk-python',
-    version='1.7.4',
+    version='1.7.5',
```
Comment (hanwavefront) [inline setup.py:18]: also update the version number in open_source_license.txt

File: README.md (hunk 1 above)
Comment (joannak-vmware) [reply README.md:212]: Meant to remove. This is no longer in the WavefrontProxyClient. Will make changes to comment

**Review by hanwavefront — APPROVED** [review]

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**